src/components/Events.tsx

```diff
@@ -1,44 +1,44 @@
-export const mockEvents = [
-  {
-    id: 1,
-    name: "Vendy",
-    sport: "Basketball",
-  },
-  {
-    id: 2,
-    name: "Lukas",
-    sport: "Football",
-  },
-  {
-    id: 3,
-    name: "Honza",
-    sport: "Tennis",
-  },
-  {
-    id: 4,
-    name: "Vendy",
-    sport: "Box",
-  },
-  {
-    id: 5,
-    name: "Lukas",
-    sport: "Skydiving",
-  },
-  {
-    id: 6,
-    name: "Johana",
-    sport: "Jogging",
-  },
-];
+import { useState, useEffect } from "react";
 
-export const Events: React.FC = () => (
-  <>
-    {mockEvents.map(({ id, sport, name }) => (
-      <article key={id}>
-        <p>{sport}</p>
-        <p>od: {name}</p>
-        <button>Otevřít</button>
-      </article>
-    ))}
-  </>
-);
+interface Event {
```

Comment:
Let's use types https://www.youtube.com/watch?v=zM9UPcIyyhQ :D I find them simpler. I also hate anything that reminds me of object-oriented programming :D